apps/wing-console/console/app/test/describe.ts

```diff
@@ -36,8 +36,6 @@ export const describe = (wingfile: string, callback: () => void) => {
 
     await page.goto(`http://localhost:${server.port}/`);
 
-    await page.getByTestId("loading-overlay").waitFor({ state: "hidden" });
-
```

Comment:
@polamoros This part does not seem to work properly after https://github.com/winglang/wing/pull/3964. Removing it seems to work fine, WDYT?

Reply:
It should work without that. it was just a trick to double check that the Console ui was loaded.